environment.yml

```diff
@@ -6,4 +6,4 @@ dependencies:
   - python>=3.6,<3.8
   - pip
   - pip:
-      - -r requirements.txt
+      - -e .
```

Comment:
Maybe better use `-e .[dev]`? In that case, the install procedure for contributors using Conda would just become:
```bash
conda env create
conda activate tw
```
which previously would have been
```bash
conda env create
conda activate tw
pip install -e .[dev]
```

But note that this assumes anyone who uses Conda is also interested in contributing (i.e. the dev requirements). And this is not necessarily the case. Could especially be a problem for #53.

Reply:
hmm i would keep it optional to install the dev requirements. So you can have the run requirements be part of the dev requirements to do it all in one swoop. But im not sure how this is handled in conda

Reply:
Ok, then I leave it like this (it's also the described workflow in the install/contribute pages)